### PR TITLE
Chore: Explicitly set state at start, and reset on fileid change

### PR DESCRIPTION
**Pull request by JustinHoldstock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #338

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ContentSidebar.js [hunk 1]
```diff
@@ -138,6 +158,9 @@ class ContentSidebar extends PureComponent<Props, State> {
             requestInterceptor,
             responseInterceptor
         });
+
+        // Clone initial state to allow for state reset on new files
+        this.initialState = cloneDeep(this.state);
```
Comment (priyajeet) [inline src/components/ContentSidebar/ContentSidebar.js]: No need to cloneDeep, {...this.state}

File: src/components/ContentSidebar/ContentSidebar.js [hunk 2]
```diff
@@ -206,7 +229,7 @@ class ContentSidebar extends PureComponent<Props, State> {
         const hasFileIdChanged = newFileId !== fileId;
 
         if (hasFileIdChanged) {
-            this.setState({});
+            this.setState(this.initialState);
```
Comment (priyajeet) [inline src/components/ContentSidebar/ContentSidebar.js]: Same

**Pull request merged**